tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -232,6 +233,9 @@ function kbs_k8s_deploy() {
 	[ -n "$ingress" ] && _handle_ingress "$ingress"
 
 	echo "::group::Deploy the KBS"
+	if [ "${KATA_HYPERVISOR}" = "qemu-tdx" ]; then
+		export DEPLOYMENT_DIR=custom_pccs
```

Comment:
By the moment we do this, we need to have a `sgx_default_qcnl.conf` under the `custom_pccs` dir that works as expected.

That file should have the following content:
```
{
  "collateral_service": "https://api.trustedservices.intel.com/sgx/certification/v4/",
  "pccs_url": "https://localhost:8081/sgx/certification/v4/",

  // To accept insecure HTTPS certificate, set this option to false
  "use_secure_cert": false
}
```

Reply:
@fidencio changes applied